daprdocs/content/en/reference/api/placement_api.md

```diff
@@ -0,0 +1,64 @@
+---
+type: docs
+title: "Placement API reference"
+linkTitle: "Placement API"
+description: "Detailed documentation on the Placement API"
+weight: 1200
+---
+
+Dapr has an HTTP API `/placement/state` for placement service that exposes placement table information. The API is exposed on the sidecar on the same port as the healthz. This is an unauthenticated endpoint, and is disabled by default. 
+
+You need to set `DAPR_PLACEMENT_METADATA_ENABLED` environment or `metadata-enabled` command line args to true to enable it. If you are using helm, set `dapr_placement.metadataEnabled` to true.
```

Comment:
Actually this needs more detail. 
@Taction
1) Is this really just called "metadata-enabled" and not ""placement-metadata-enabled""
2) Also `metadata-enabled` needs to be added to the https://docs.dapr.io/reference/arguments-annotations-overview/ . When you say command line is this on daprd or the CLI

Reply:
This is the environment or command line args for the `placement` binary not for daprd or cli, and this is supposed to only be used in self-host mode(In k8s mode, the helm value should be used). I've found this doc https://docs.dapr.io/operations/hosting/self-hosted/self-hosted-no-docker/#enable-actors  introduces how to run placement manually, maybe add a sub-section for supported  `arguments and environments` here.

Reply:
I see this was not clear. I have made a suggestion below for an update.